### ISSUE TITLE
[Bug] I need logs)

### ISSUE THREAD

**Issue by Savranchuk** [Rapha149/ArmorStandEditor#9]
### Plugin version

1.5.7

### Server software and version

purpur 1.21

### Steps to reproduce

Using the plugin, you can take the armor stand into your inventory and this is not logged in any way.

### Expected behaviour

We need to somehow make armor stand logs

### What is actually happening?

The player took the armor stand and it is impossible to find who did it

**Comment by Rapha149** [comment]
What are racks?

**Comment by Savranchuk** [comment]
racks - armorstand*

**Comment by Rapha149** [comment]
So is your problem that players can do this or that it isn't logged?

**Comment by Savranchuk** [comment]
Right. The owners of the armorstand forget to make it private and someone else's player comes and breaks the armorstand or takes things from it. I need logs to be written somehow

**Comment by Rapha149** [comment]
Well that's the default behavior for armor stands, without the plugin players could break it or take items from it, too.
